codacy/values.yaml

```diff
@@ -339,71 +269,6 @@ codacy-ingress:
           servicePort: http
         path: /monitoring
 
-codacy-api:
-  replicaCount: 1
-  fullnameOverride: codacy-api
-  config:
-    # Trial license. Valid for 4 users until the end of 2020
-    license: "RAjlCYb/DxsAME0TjIn55neFSYOt0yxxnXBh4nKe/hbz4mjr0dDs3TLOehzQ3QyaLfONRva7T2iSkA57bMdpI/ZwAOUQtjO5XiU1KDaSUXCFZwDu1mBJMlMONCVOV3AuB6y56tsL0CZXnBqZD3R9dShTVXiSYDt/+2dhlVqGb70="
-    play:
-      configFile: "enterprise.conf"
-      logFile: logback.xml
-  metrics:
-    serviceMonitor:
-      enabled: false
-    grafana_dashboards:
-      enabled: false
-
-worker-manager:
-  replicaCount: 1
-  fullnameOverride: codacy-worker-manager
-  config:
-    play:
-      configFile: enterprise.conf
-      logFile: logback.xml
-    cloudAmqp:
-      workerUser: "rabbitmq"
-      workerPass: "rabbitmq"
-    workers:
-      dedicatedMax: 5
-      inactivityTimeout: 30
-      analysis:
-        # Ignore files with size bigger than this value during analysis
-        maxFileSizeBytes: 150000
-      # Toleration and nodeSelector for worker/tool pods
-      tolerations: []
-      nodeSelector: {}
-
-    codacy:
-      worker:
-        imageVersion: stable
-
-    # The resources to assign to a worker pod
-    workerResources:
-      requests:
-        cpu: 1
-        memory: 1Gi
-      limits:
-        cpu: 1
-        memory: 2Gi
-
-    # The resources to assign to a tool pod
-    pluginResources:
-      requests:
-        cpu: 0.5
-        memory: 1000000000 # this value needs to be set in bytes
-      limits:
-        cpu: 1
-        memory: 3000000000 # this value needs to be set in bytes
-
-codacy-spa:
-  replicaCount: 1
-  fullnameOverride: codacy-spa
-  config:
-    codacy:
-      features:
-        passwordLogin: true
```

Comment:
this is already set in the chart itself:
https://github.com/codacy/codacy-spa/pull/179/files#diff-c7eecabc2da679d337f890d31b076513L4